### ISSUE TITLE
[Broken Website] Hifi Compass

### ISSUE THREAD

**Issue by ShalokShalom** [darkreader/darkreader#11986]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Unreadable Charts

### Website Address

https://hificompass.com/en/speakers/measurements/ciare/ciare-hwb200

### Steps To Reproduce

1 Go on the website
2 Click on **Axial magnitude frequency response** or any of the entries below it.
3 Notice how the rest of the homepage is ugly, mostly due to the brown colors

### Screenshots

![Screenshot_20231208_225256](https://github.com/darkreader/darkreader/assets/6344099/699e043b-cba8-4ea9-805f-73f615f33342)
![Screenshot_20231208_225246](https://github.com/darkreader/darkreader/assets/6344099/6afe81ff-bea5-4ce7-824b-ef59e783d15d)

### Operating System

Linux

### Web Browser name and version

Brave

### Dark Reader version

4.9.73

**Comment by Myshor** [comment]
All is related to not default contrast setting. When default is used the chart is dark but readable.

To fix this I force background like it is used originally without dark reader. (so it will be bright)

To check the fix you need to have "Synchronize site fixes" option enabled, which is available in new design Settings => Manage Settings.
![obraz](https://github.com/darkreader/darkreader/assets/56877029/39791691-bf5f-49f8-b35a-a8fbf2c891d4)

To swich design you need to open Developer Tools and click the button to switch to new design.
![obraz](https://github.com/darkreader/darkreader/assets/56877029/259ffb9b-9688-431d-850d-82596828b62d)

**Comment by ShalokShalom** [comment]
So can we automate this? It seems like something that vast majority of users would not know how to do, and they will likely care not to google for this very issues. 

I imagine. 😕 

Thanks for your help!

**Comment by Myshor** [comment]
Not that easy to automate... that chart pictures has transparent backgrounds so when Dark Reader was changing background color for the site then the background for charts was changed. In some cases picture will look ok with dark background in other it will not. When it's not looking fine the the manual fixes can be added like I added in for this site.

And again I will repeat that the issue you found was because very big contrast setting. I almost answered that I can not reproduce the issue then I saw non standard settings for Dark Reader.

To end my talk: We won't set all transparent pictures to white (or almost white) background because in most cases they look fine with dark background.
